src/PKSim.Infrastructure/Reporting/Markdown/MarkdownTracker.cs

```diff
@@ -1,20 +1,22 @@
-﻿using System.Text;
+﻿using System.Linq;
+using System.Text;
 using MarkdownLog;
+using OSPSuite.Core.Extensions;
+using OSPSuite.Utility.Extensions;
 
 namespace PKSim.Infrastructure.Reporting.Markdown
 {
    public class MarkdownTracker
    {
+      private const string OFFSET = "     ";
+      private const string TABLE_OFFSET = "\r\n     ";
```

Comment:
This is kind of weird. The lib creates some offset that neither VSCOde nor github see as valid markdown. Removing it manually